hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/ConfigurationKeys.java

```diff
@@ -69,6 +69,9 @@
    *  Default is empty. **/
   public static final String FS_AZURE_APPEND_BLOB_KEY = "fs.azure.appendblob.directories";
   public static final String FS_AZURE_READ_AHEAD_QUEUE_DEPTH = "fs.azure.readaheadqueue.depth";
+  public static final String FS_AZURE_ALWAYS_READ_BUFFER_SIZE = "fs.azure.always.read.buffer.size";
```

Comment:
Nit: maybe call it sth like `fs.azure.always.read.buffer.size.enabled` since it is a boolean flag. The "size" suggests it is an integer.

Reply:
> Nit: maybe call it sth like `fs.azure.always.read.buffer.size.enabled` since it is a boolean flag. The "size" suggests it is an integer.

Thanks. Will incorporate this change into the Apache hadoop PR.